.github/workflows/add_to_project.yml

```diff
@@ -0,0 +1,20 @@
+name: Add bugs to bugs project
+
+on:
+  issues:
+    types:
+      - opened
+
+jobs:
+  add-to-project:
+    name: Add issue to project
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/add-to-project@RELEASE_VERSION
```

Comment:
You probably want to specify the release version here. Latest release is v1.0.1. Set it to v1 or v1.0 perhaps?

https://github.com/actions/add-to-project/releases